integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableRenameCommand.scala

```diff
@@ -83,6 +85,11 @@ private[sql] case class CarbonAlterTableRenameCommand(
           sparkSession)
       carbonTable = metastore.lookupRelation(Some(oldDatabaseName), oldTableName)(sparkSession)
         .asInstanceOf[CarbonRelation].carbonTable
+      // if any load is in progress for table, do not allow rename table
+      if (SegmentStatusManager.checkIfAnyLoadInProgressForTable(carbonTable)) {
+        throw new AnalysisException(s"alter rename failed, load, insert or insert " +
```

Comment:
use operationNotAllowed

Reply:
Since this PR not just modify for drop table command, can you add the reject logic in all impacted command. I think  all commands that involves metadata modification is not allowed, right? @ravipesala please have a look

Reply:
@jackylk yes, we have identified drop, alter,and delete segment commands which are impacted, so we need to add restriction to them.